roles/os_hardening/defaults/main.yml

```diff
@@ -390,3 +390,22 @@ os_selinux_enabled: true
 # The values must be inside the 1000-999999999 range.
 os_sha_crypt_min_rounds: "640000"
 os_sha_crypt_max_rounds: "640000"
+
+# Define /tmp permissions
+tmp_dir_mode: '1777'
+
+# Define /tmp is a mountpoint.
+# If it is not, harden /tmp folder permissions
+tmp_mnt_enabled: false
+
+# Define source mountpoint for folder /tmp
+# example: /dev/vg_system/lv_tmp
+tmp_mnt_src: ""
+
+# Define filesystem for tmp mount
+# default: ext4
+tmp_mnt_filesystem: "ext4"
+
+# Mount options for /tmp in /etc/fstab.
+tmp_mnt_options: 'rw,nosuid,nodev,noexec,relatime'
```

Comment:
@rndmh3ro should we keep the `os_` prefix for sake of consistency here?

Reply:
Yes, good idea.

Reply:
@rndmh3ro @schurzi 
so should I add 'os_' before any variable? Alternatively I can rename it, if you prefer this way more?

Reply:
Ad `os_`, please.